packages/now-cli/src/util/dns/get-dns-records.ts

```diff
@@ -40,12 +48,16 @@ function getAddDomainName(domainNames: string[]) {
     ...prev,
     {
       domainName: domainNames[idx],
-      records: item.sort((a, b) => a.slug.localeCompare(b.slug)),
+      records: item,
     },
   ];
 }
 
-async function getDomainNames(client: Client, contextName: string) {
-  const { domains } = await getDomains(client, contextName);
-  return domains.map(domain => domain.name).sort((a, b) => a.localeCompare(b));
+async function getDomainNames(
+  client: Client,
+  contextName: string,
+  next?: number
+) {
+  const { domains, pagination } = await getDomains(client, contextName, next);
```

Comment:
Do you need to bump the API version here to support pagination?

Reply:
@styfle `getDomains` already returns paginated domains :)